src/main/java/frc/robot/Robot.java

```diff
@@ -74,47 +74,60 @@ public void configureButtonBindings() {
                 .whenReleased(shooter.stop());
 
         // Intake:
-        driveController.a().whenPressed(intake.extend(SpinDirection.COUNTERCLOCKWISE))
-                .whileHeld(ballTransport.loadCargoWithIntake())
-                .whenReleased(sequence("Ball transport end",
-                        race("Finish Transport", new WaitCommand(2.5),
-                                ballTransport.loadCargoWithIntake()),
-                        parallel("Intake retracted w/ Ball Transport",
-                                ballTransport.stopTransport(), intake.retract())));
+        // These commands are duplicated for both the drive and copilot controllers.
+        final CommandBase intakeExtend = intake.extend(SpinDirection.COUNTERCLOCKWISE);
+        final CommandBase loadBall = ballTransport.loadCargoWithIntake();
+        final CommandBase intakeRetract = sequence("Ball transport end",
```

Comment:
This may cause problems if both driver and copilot attempt to use the command at the same time. We should definitely test this.